plugins/chat/assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -53,14 +53,8 @@ export default class ChatLivePane extends Component {
   @tracked needsArrow = false;
   @tracked loadedOnce = false;
 
-  isAtBottom = true;
-  isTowardsBottom = false;
-  isTowardsTop = false;
-  isAtTop = false;
-
```

Comment:
This all has been changed to DOM functions instead of compute properties which are too slow for something which is so much in the hot path